Other rules improvements (#4)
raunc/eutpop-ref-sheet: src/other_rules.tex

\sectionbg{Other Rules}

\subsection*{General Notes \rulesref{(p.~2-4)}}
\subsubsection*{Concepts \& Terms}
\begin{itemize}

> 	\item \strong{Adjacent}:

Reviewer: ```suggestion
	\item \strong{Adjacency}:
```

	\begin{itemize}
		\item A Prov./Unit/token is adj. to their Area, to ones sharing a border with it and to Prov. in them
		\item A Port is adj. also to its Sea Zones, to Ports facing them and to their Areas
		\item Areas/Sea Zones are adj. if they share a border or have Ports facing the same Sea Zone

> 		\item A Realm is adj. if an Own Prov. is adj.

Reviewer: ```suggestion
		\item A \strong{Prov.}/\strong{Unit}/\strong{token} is Adj. to their Area, to ones sharing a border with it and to Prov. in them
		\item A \strong{Port} is Adj. also to its Sea Zones, to Ports facing them and to their Areas
		\item \strong{Areas}/\strong{Sea Zones} are Adj. if they share a border or have Ports facing the same Sea Zone
		\item A \strong{Realm} is Adj. if an Own Prov. is Adj.
```
Let's highlight the terms which adjacency is defined.
Also "Adjecent" is a game term, so let's capitalize.

	\end{itemize}

> 	\item \strong{Deployed}: of a Unit on the board or in an Army/Fleet

Reviewer: ```suggestion
	\item \strong{Deployed}: Unit on board or in Army/Fleet
```
Fits on 1 line like this

	\item \strong{Friendly}:
	\begin{itemize}
		\item Armies/Units/Prov. Controlled by you/\ally/\vassal
		\item Non-Hostile Areas containing Friendly Prov.
	\end{itemize}
	\item \strong{Hostile}:
	\begin{itemize}

> 		\item Armies/Units/Prov. Controlled by \rebels/Realms at War with you and Areas

Reviewer: ```suggestion
		\item Armies/Units/Prov. Controlled by \rebels/Realms at War with you
```

		\item Areas containing Hostile Units/Prov.
		\item Sea Zones containing Hostile Ships
		\item Sea Zones facing Hostile NPR Ports \conj{and} containing no Ship Hostile to them
	\end{itemize}
	\item \strong{Neutral}: neither Friendly nor Hostile
	\item \strong{Opponent}: player other than you
	\item \strong{Player}: human or Bot
	\item \strong{You}: the Active Player

> 	\item \strong{Occupied Province}: has Occ. token \conj{or} 2 \rebel/\town/\vassal (Controller on top, Lawful Owner on bottom)

Reviewer: ```suggestion
	\item \strong{Occupied Province}: has Occ. token \conj{or} 2 \rebeltown/\town/\vassal (Controller on top, Lawful Owner on bottom)
```

	\item \strong{Own Province}: has its Realm's flag/\town/\dnpr \conj{and} is non-Occupied
	\item \strong{Realm}: made of Own Prov. and Areas with such Prov.
	\begin{itemize}
		\item \strong{(N)PR}: (Non-)Player Realm
		\item \strong{Distant R.}: with Dist. Capital
		\item \strong{R. Core Provinces}: has its flag, \conj{or}, superseding, its \core in the Prov. Area
	\end{itemize}
	\item \strong{Colonial Claim}: \claim on vacant Territory
\end{itemize}

\subsubsection*{Token Limits}
\begin{itemize}
	\item Generally limited. If nothing in supply, take from anywhere. Exceptions below
	\item \strong{May not be moved}
	\begin{itemize}
		\item Towns (\town), Vassal tokens (\vassal)
		\begin{itemize}
			\item If out of Large \towns, use 2 Small \towns instead
			\item If out of Small \towns, use a \vassal instead

> 		\end{itemize}

Reviewer: This is a nice place for them. I'd also like to add a reference to p. 21 here, but no good idea how to fit it. Perhaps merging those two points into 1 and adding the reference to the end would work. Or maybe you have a better idea?

Author: I tried by separating Towns from Vassals, otherwise merging the two points is ok too.

	\end{itemize}
\end{itemize}

> \subsubsection*{Bankruptcy}

Reviewer: I would keep/add these references when parent heading references multiple pages and there is enough room. Then it is easier to find the exact place in rulebook if needed.

Except, probably not helpful for the ones where info comes from multiple pages anyway (e.g Concepts & Terms above)

	\item Removed if Prov. is Annex. or Vassalized
\end{itemize}

\subsection*{Rebels (\rebel) \rulesref{(p.~37)}}
\subsubsection*{Rebel Dice}
\begin{itemize}
	\item Roll Area by Area

> 	\item Roll 1 dice per \unrest you have in a given Area

Reviewer: ```suggestion
	\item Roll 1 die per \unrest you have in a given Area
```
Singular of "dice" is "die"

		\item Have access to Papal Curia
		\item May \action{Change State Religion} to Protestantism from Age II onwards
	\end{itemize}

> 	\item The Papal State is always Catholic (p.~5)

Reviewer: ```suggestion
	\item The \guilsinglright Papal States is always Catholic (p.~5)
```
Let's write Realm names with a preceding arrow like in the rulebook and cards

	\item Bots never counter Covert Actions 
\end{itemize}

\subsectionbot{Bot Warfare \botrulesref{(p.~4-5)}}
\begin{itemize}

> 	\item Bot deploys always in its Army

Reviewer: ```suggestion
	\item Bot recruits always into its Army
```
